scipy/special/tests/test_basic.py

```diff
@@ -3716,6 +3717,31 @@ def test_riccati_yn(self):
         assert_array_almost_equal(C, special.riccati_yn(n, x), 8)
 
 
+class TestSoftplus:
+    @pytest.mark.parametrize("value, expected",[
+        (0, 0.6931471805599453),
+        ([-1, 0, 1], np.array([0.31326169, 0.69314718, 1.31326169])),
+        (100, 100.0),
+        (-5, 0.006715348489118068),
+        (-100, 3.720075976020836e-44),
+        (10000, 10000.0)
+    ])
```

Comment:
Good idea to test all those cases, but this doesn't quite do that:

![image](https://github.com/user-attachments/assets/00755e94-3fa3-4050-a177-408594a639a8)

Namely, `18 < x < 33.3` seems to be missing. Also the source of the reference values is not provided. I'll update accordingly.